tests/Unit/blocks/recommendations/block-structure.test.js

```diff
@@ -0,0 +1,342 @@
+/**
+ * External dependencies.
+ */
+import { render, screen, within } from '@testing-library/react';
+import '@testing-library/jest-dom';
+
+/**
+ * Internal dependencies.
+ */
+import ParselyRecommendations from '../../../../src/blocks/recommendations/components/parsely-recommendations';
+import ParselyRecommendationsList from '../../../../src/blocks/recommendations/components/parsely-recommendations-list';
+import ParselyRecommendationsTitle from '../../../../src/blocks/recommendations/components/parsely-recommendations-title';
+import RecommendationsStore from '../../../../src/blocks/recommendations/recommendations-store';
+
+/**
+ * Initializations.
+ */
+const apiResultSingle = [
+	{
+		author: 'John Doe',
+		authors: [ 'John Doe', 'Jane Doe' ],
+		full_content_word_count: 0,
+		image_url: 'https://example.com/image.jpg',
+		metadata: null,
+		pub_date: '2022-03-14T13:14:00',
+		section: 'Articles',
+		tags: [ 'parsely_smart:entity:Tag 1', 'parsely_smart:iab:Tag 2' ],
+		thumb_url_medium: 'https://example.com/image-thumb.jpg',
+		title: 'My nice article',
+		url: 'https://example.com/my-nice-article/',
+	},
+];
+const apiResultMultiple = [
```

Comment:
Maybe we could construct this one doing something like:

```
const apiResultMultiple = apiResultSingle.concat([
  // new items
  ...
])
```

Reply:
If you think this is better, I can go with that. In that case I'll have to modify some checks though because the two objects aren't the same. Let me know what you would prefer.

Reply:
I misread and thought they were identical. Ignore my comment.